v2.0/backup.md

```diff
@@ -6,7 +6,7 @@ toc: false
 
 {{site.data.alerts.callout_danger}}The <code>BACKUP</code> feature is only available to <a href="https://www.cockroachlabs.com/product/cockroachdb/">enterprise</a> users. For non-enterprise backups, see <a href="sql-dump.html"><code>cockroach dump</code></a>.{{site.data.alerts.end}}
 
-CockroachDB's `BACKUP` [statement](sql-statements.html) creates full or incremental backups of your cluster's schemas and data that are consistent as of a given timestamp. These backups can be stored on the platforms you're already using, including AWS S3, Google Cloud Storage, NFS, or HTTP storage.
+CockroachDB's `BACKUP` [statement](sql-statements.html) allows you to create full or incremental backups (with or without revision history) of your cluster's schema and data that are consistent as of a given timestamp.
```

Comment:
nit: This feels a bit overloaded. It'd be easy to split this into 2 sentences plus add a link to the revision history section, e.g.:

> CockroachDB's `BACKUP` [statement](sql-statements.html) allows you to create full or incremental backups of your cluster's schema and data that are consistent as of a given timestamp. Backups can be with or without [revision history](#backups-with-revision-history) (**new in v2.0**).